tests/components/template/test_binary_sensor.py

```diff
@@ -15,7 +15,7 @@
 from homeassistant.core import CoreState
 import homeassistant.util.dt as dt_util
 
-from tests.common import assert_setup_component, async_fire_time_changed
+from tests.common import async_fire_time_changed
```

Comment:
This file is just whitespace changes: https://github.com/home-assistant/core/pull/48273/files?w=1

I removed `assert_setup_component` which is 🤷